scripts/flows/entry_workflows/main_cid_ingestion.py

```diff
@@ -45,18 +43,3 @@ def cid_ingestion_pipeline(target_url: str = None, target_key: str = None):
         settings_dict,
         minio_client
     )
-
-
-def build_deployment():
```

Comment:
Yup, like you centralized it.

Reply:
This seemed like the cleaner solution. I also created a single make command to be able to call the ingests from the cli.